content/en/case-studies/appdirect/index.html

```diff
@@ -1,15 +1,11 @@
 ---
 title: AppDirect Case Study
-
 linkTitle: AppDirect
 case_study_styles: true
 cid: caseStudies
 css: /css/style_case_studies.css
-logo: appdirect_featured_logo.png
+logo: appdirect_featured_logo.svg
 featured: true
-weight: 4
-quote: >
-   We made the right decisions at the right time. Kubernetes and the cloud native technologies are now seen as the de facto ecosystem.
 ---
 
 <div class="banner1" style="background-image: url('/images/CaseStudy_appdirect_banner1.jpg')">
```

Comment:
@alexcontini would it be OK to separate this fix into its own pull request? The PR title implies that this PR only touches the Denso case study.